### PR TITLE
Update typescript docs

### PR DESCRIPTION
**Pull request by dac09**
Updates docs for setting up typescript - no longer manual! 🎊

### CODE REVIEW

File: docs/cliCommands.md [hunk 1]
```diff
@@ -401,10 +401,10 @@ For the full list of Webpack Dev Server settings, see [this documentation](https
 
 Deploy your redwood project to a hosting provider target.
 
-**Netlify, Vercel, and Render**  
+**Netlify, Vercel, and Render**
 For hosting providers that auto deploy from Git, the deploy command runs the set of steps to build, apply production DB changes, and apply data migrations. In this context, it is often referred to as a Build Command. _Note: for Render, which uses traditional infrastructure, the command also starts Redwood's api server._
 
-**AWS**  
+**AWS**
 This command runs the steps to both build your project _and_ deploy it to AWS.
```
Comment (thedavidprice) [inline docs/cliCommands.md:408]: Without the two spaces, this collapses from a header into the first sentence. I think that would be fine if you added a ":", e.g. `Netlify, Vercel, and Render:`

Here's what it looks like with this edit:
Screen Shot 2021-04-27 at 9 18 00 AM

File: docs/typescript.md [hunk 2]
```diff
@@ -2,72 +2,48 @@
...
-    "paths": {
-      "src/*": ["./src/*"]
-    },
-    "typeRoots": ["../node_modules/@types", "../.redwood"],
-    "types": []
-  },
-  "include": ["src", "../.redwood/globals"]
-}
+1. Put your shared types at the root of the project (makes sense right?), maybe in `types` at the root
+2. You can call this folder whatever you like ofcourse, just add the same folder to your includes
```
Comment (thedavidprice) [inline docs/typescript.md:32]:
```suggestion
2. You can call this folder whatever you like, of course, just add the same folder to your includes
```

File: docs/typescript.md [hunk 3]
```diff
@@ -78,4 +54,33 @@ If you're adding tests, you'll want to include the types for `jest` in your `tsc
...
+1. Create a `./api/tsconfig.json` file:
+
+```shell
+touch api/tsconfig.json
+```
+
+2. Now copy and paste the latest config from the Redwood template: [api/tsconfig.json](https://github.com/redwoodjs/redwood/blob/main/packages/create-redwood-app/template/api/tsconfig.json) into the empty file
+
+### WEB
+
```
Comment (thedavidprice) [inline docs/typescript.md:72]:
```suggestion
```

**Review by thedavidprice — CHANGES_REQUESTED** [review]
Looking good! A few minor edits to handle.

File: docs/cliCommands.md (hunk 1 above)
Comment (dac09) [reply docs/cliCommands.md:408]: Yeah it was the auto formatter I think. Nice catch, I'll add a <br>

Comment (dac09) [reply docs/cliCommands.md]: Done ✅

**Review by thedavidprice — APPROVED** [review]

**Pull request merged**